data/homo-sapiens-citation-information.tsv

```diff
@@ -0,0 +1,11 @@
+#name	chromosome	start	length	color	full_name	citations_from_2021_02_10_to_2021_07_15	significance

```

Comment:
Future refactoring idea: It might be worth changing this field into multiple fields like "date_range_start" and "date_range_end" to make it easier to use.